help/grandorgue.xml

```diff
@@ -7406,7 +7406,7 @@ Has the following attributes:
           <term>DispShortcutKeyLabelFont</term>
           <listitem>
             <para>
-(string, required) Name of the font for keyboard labels
+(string, required) Name of the font for shortcut labels
```

Comment:
I don't know if it should be mentioned in the help or not, but I don't think that GO use this feature at all (and as far as I recall never has) since the keyboard shortcuts aren't displayed next to any items that have them set. Perhaps it wasn't implemented in the displaymetrics, but I think that it dates all the way back to the MyOrgan time (which didn't display them either as I remember).

Reply:
But this feature actually existed in HW1, which is why it still exist in the ODF as it's a valid entry in such sample sets.

Reply:
I'll add "provided just for HW1 compatibility. DEPRECATED." like `StopControlMIDIKeyNumber` and `MIDIProgramChangeNumber`.

Reply:
@willeke1234 That's fine with me.